dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/services/CompanyRolesManager.kt

```diff
@@ -17,10 +17,9 @@ import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.stereotype.Service
 import org.springframework.transaction.annotation.Transactional
 import java.util.*
-
+// TODO question to the reviewer should @params be added to the documentation?
 /**
```

Comment:
I'm for @params without strong feelings/tendencies, but more importantly, I'm for consistency.